hydra/periodicmetrics.go

```diff
@@ -0,0 +1,55 @@
+package hydra
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/ipfs/go-datastore/query"
+	"github.com/libp2p/hydra-booster/metrics"
+	"go.opencensus.io/stats"
+)
+
```

Comment:
why not have the periodicmetrics.go in the metrics folder?

Reply:
I put them here because, while _these_ metrics are recorded periodically there are others that are recorded in realtime. The metrics recorded in realtime cannot all live in the `metrics` package, and the fact that we record some periodically is an artifact of performance concerns and accessibility of the data. We'd ideally record them in realtime if we could, and if we did, they'd live in the hydra package.

So, rather than split metrics recording between different packages I chose to keep recorded metrics close to the thing they are recording metrics _for_ and I reserved the `metrics` package for the server setup and definitions of _what_ metrics can be collected.